LFPy/recextelectrode.py

```diff
@@ -176,8 +173,15 @@ def __init__(self, cell=None, sigma=0.3,
                  seedvalue=None, **kwargs):
         """Initialize RecExtElectrode class"""
 
-        self.cell = cell
         self.sigma = sigma
+        if type(sigma) in [list, np.ndarray]:
+            self.sigma = np.array(sigma)
+            self.anisotropic = True
+        else:
+            self.sigma = sigma
+            self.anisotropic = False
+
+
```

Comment:
I think here would be a good idea to assert `sigma.shape == (3, )` if we have anisotropy. Also, in the class docstring say that sigma can be either a float or len 3 tuple/list/ndarray